tests/examples_methods_syntax/isotype.py

```diff
@@ -69,7 +69,7 @@
 alt.Chart(source).mark_point(filled=True, opacity=1, size=100).encode(
     alt.X('x:O').axis(None),
     alt.Y('animal:O').axis(None),
-    alt.Row('country:N').header(title=''),
+    alt.Row('country:N').title(''),
```

Comment:
```suggestion
    alt.Row('country:N').title(None),
```
I prefer `None` over `''`.

Reply:
This is interesting. I think `None` is more semantically correct but `''`  is more convenient to type and I remember that when I started to use Altair a few years ago, there was some context where `None` didn't work, but `''` did. It is possible that I was just doing something wrong back then, but I will see if I can find it again.

Reply:
Might be, I can remember something similar. But I would consider that as a limitation of Vega-Lite at that moment.